services/housetables/src/main/java/com/linkedin/openhouse/housetables/api/validator/impl/OpenHouseUserTableHtsApiValidator.java

```diff
@@ -60,10 +68,15 @@ public void validateGetEntities(UserTable userTable) {
     }
 
     if (userTable.getTableId() != null
```

Comment:
Can 70 to 79 be combined?

if (userTable.getTableId() != null) {
check databaseId not null;
check for pattern;
}

Reply:
Sure.